src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,116 @@
+package core.basesyntax.service;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final String DEFAULT_EXCEPTION = RegistrationException.class.toString();
+    private static final String DEFAULT_LOGIN = "validUser";
+    private static final String DEFAULT_PASSWORD = "Qwertyu";
+    private static final String MIN_LENGTH_PASSWORD = "Qwerty";
+    private static final String INVALID_PASSWORD = "Qwert";
+    private static final int DEFAULT_AGE = 19;
+    private static final int MIN_AGE = 18;
```

Comment:
```suggestion
    private static final int MIN_AGE = 18;
```
you can remove this constant at all and use MIN_AGE constant instead of it everywhere